README.md

```diff
@@ -13,6 +13,89 @@ Kolibri Studio supports the following workflows:
 Kolibri Studio uses Django for the backend and Backbone.js for the frontend.
 
 
+## Getting a local preview of Studio
+
+One of Kolibri Studio's deployment methods use Kubernetes and Docker. If you want a 
+quick and easy way to get Studio up and running with minimal configuration,
+this is the method you want to use. This works best on a Linux or MacOS machine.
+
+## Prerequisites
+
+Download the following applications on to your local machine, in order:
+
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/), the program 
+to control Kubernetes clusters.
+
+- [Docker](https://docs.docker.com/install/), the containerization platform LE uses.
+
+- [ksonnet](https://ksonnet.io/#get-started), an abstracting application on top
+  of kubectl to make deployments easier.
+
+- [Minikube](https://github.com/kubernetes/minikube#installation), an 
+application to get a local Kubernetes cluster up and running.
+
```

Comment:
`minikube start ...` command seems to require VirtualBox (got `E0420 VBoxManage not found. Make sure VirtualBox is installed and VBoxManage is in the path`) so maybe add something like:

     - [VirtualBox](https://www.virtualbox.org/wiki/Downloads), virtual machine emulator required for Minikube.